### PR TITLE
Added controller manager secret

### PR DESCRIPTION
**Pull request by Raffo**
Depends on webhook change, merged.

### CODE REVIEW

File: cluster/userdata-master.yaml [hunk 1]
```diff
@@ -337,7 +337,7 @@ write_files:
             limits:
               cpu: 200m
               memory: 1Gi
-        - image: registry.opensource.zalan.do/teapot/k8s-authnz-webhook:v0.1.4
+        - image: registry.opensource.zalan.do/teapot/k8s-authnz-webhook:v0.1.4-dirty-3
```
Comment (szuecs) [inline cluster/userdata-master.yaml]: dirty ?

Comment (Raffo) [inline cluster/userdata-master.yaml]: It's just a placeholder, the new image will be ready as soon as the dependent PR is merged.

**Comment by szuecs** [comment]
:+1:

**Comment by Raffo** [comment]
👍

**Pull request merged**